code/modules/ruins/spaceruin_code/hilbertshotel.dm

```diff
@@ -538,3 +568,10 @@ GLOBAL_VAR_INIT(hhmysteryRoomNumber, 1337)
 	Lay your head to rest, it soon becomes clear<br>
 	There's always more room around every bend<br>
 	Not all that's countable has an end...<i>"}
+
+
+/obj/item/paper/crumpled/hilbertsstaffnotice
+	name = "An apology leter to our guests"
+	info = {"Due to the bluespace technology used to operate the door to our hotel being unstable as of late, we have modified them to allow an emergency escape option.<br>
+	Each room also recieved the equipment necessary to ensure a comforatble travel back home.<br>
+	The staff of Hilbert's Hotel deeply apologizes for the inconvenience and assures our guests that our best technicians are working on solving the issue.<br>"}
```

Comment:
Spelling mistakes.
Also I think this should mention an incident involving someone pulling the doorhandle and damaging it causing bluespace instabilities that have been resolved. Damaging the door handle may result in a forced ejection of personnel from their rooms.

Reply:
Engrish is hawd, okay